options/js/options.js

```diff
@@ -1,29 +1,51 @@
+var recentSearchQueries = [];
+var recentSearchQueryUrls = [];
+var historyListElement;
+var clearHistory;
+var count;
+var themeToggle = document.getElementById("theme");
 
-var radios = document.getElementsByName('theme');
+if (!localStorage.getItem("theme")) localStorage.setItem("theme", "light");
```

Comment:
```
var theme = localStorage.getItem(THEME);
var isThemeNotSetInLocalStorage = !!theme; // think what could be a better name for the variable.
if ( isThemeNotSetInLocalStorage ) {
   localStorage.setItem(THEME, LIGHT);
}
```

Also we can move 

var THEME = "theme";
var LIGHT = "light";
var DARK = "dark";

to the top of file and use these constants everywhere. It is better to move all constants to the top of file.